CHANGELOG.md

```diff
@@ -6,6 +6,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased 1.3.z]
 
+## [1.3.2] - 2025-10-02
+
+> Ночь, улица, фонарь, аптека...
+
+### Changed
+ * The conversion from cgroup v1 CPU shares to cgroup v2 CPU weight is
+   improved to better fit default v1 and v2 values. (#4772, #4785, #4897)
+ * Dependency github.com/opencontainers/cgroups updated from v0.0.1 to
```

Comment:
This has already been mentioned in the previous item, so shall we remove it here?